homeassistant/components/lg_thinq/sensor.py

```diff
@@ -0,0 +1,590 @@
+        translation_key=ThinQProperty.FLAVOR_INFO,
+    ),
+    ThinQProperty.BEER_REMAIN: SensorEntityDescription(
+        key=ThinQProperty.BEER_REMAIN,
+        native_unit_of_measurement=PERCENTAGE,
+        translation_key=ThinQProperty.BEER_REMAIN,
+    ),
+}
+REFRIGERATION_SENSOR_DESC: dict[ThinQProperty, SensorEntityDescription] = {
+    ThinQProperty.FRESH_AIR_FILTER: SensorEntityDescription(
+        key=ThinQProperty.FRESH_AIR_FILTER,
+        translation_key=ThinQProperty.FRESH_AIR_FILTER,
+    ),
+    ThinQProperty.ONE_TOUCH_FILTER: SensorEntityDescription(
+        key=ThinQProperty.ONE_TOUCH_FILTER,
+        translation_key=ThinQProperty.ONE_TOUCH_FILTER,
+    ),
+}
+RUN_STATE_SENSOR_DESC: dict[ThinQProperty, SensorEntityDescription] = {
+    ThinQProperty.CURRENT_STATE: SensorEntityDescription(
+        key=ThinQProperty.CURRENT_STATE,
+        translation_key=ThinQProperty.CURRENT_STATE,
+    ),
+    ThinQProperty.COCK_STATE: SensorEntityDescription(
+        key=ThinQProperty.COCK_STATE,
+        translation_key=ThinQProperty.COCK_STATE,
+    ),
+    ThinQProperty.STERILIZING_STATE: SensorEntityDescription(
+        key=ThinQProperty.STERILIZING_STATE,
+        translation_key=ThinQProperty.STERILIZING_STATE,
+    ),
+    ThinQProperty.GROWTH_MODE: SensorEntityDescription(
+        key=ThinQProperty.GROWTH_MODE,
+        translation_key=ThinQProperty.GROWTH_MODE,
+    ),
+    ThinQProperty.WIND_VOLUME: SensorEntityDescription(
+        key=ThinQProperty.WIND_VOLUME,
+        translation_key=ThinQProperty.WIND_VOLUME,
+    ),
+}
+TEMPERATURE_SENSOR_DESC: dict[ThinQProperty, SensorEntityDescription] = {
+    ThinQProperty.TARGET_TEMPERATURE: SensorEntityDescription(
+        key=ThinQProperty.TARGET_TEMPERATURE,
+        native_unit_of_measurement=UnitOfTemperature.CELSIUS,
+        translation_key=ThinQProperty.TARGET_TEMPERATURE,
+    ),
+    ThinQProperty.DAY_TARGET_TEMPERATURE: SensorEntityDescription(
+        key=ThinQProperty.DAY_TARGET_TEMPERATURE,
+        native_unit_of_measurement=UnitOfTemperature.CELSIUS,
+        state_class=SensorStateClass.MEASUREMENT,
+        translation_key=ThinQProperty.DAY_TARGET_TEMPERATURE,
+    ),
+    ThinQProperty.NIGHT_TARGET_TEMPERATURE: SensorEntityDescription(
+        key=ThinQProperty.NIGHT_TARGET_TEMPERATURE,
+        native_unit_of_measurement=UnitOfTemperature.CELSIUS,
+        state_class=SensorStateClass.MEASUREMENT,
+        translation_key=ThinQProperty.NIGHT_TARGET_TEMPERATURE,
+    ),
+    ThinQProperty.TEMPERATURE_STATE: SensorEntityDescription(
+        key=ThinQProperty.TEMPERATURE_STATE,
+        translation_key=ThinQProperty.TEMPERATURE_STATE,
+    ),
+    ThinQProperty.CURRENT_TEMPERATURE: SensorEntityDescription(
+        key=ThinQProperty.CURRENT_TEMPERATURE,
+        native_unit_of_measurement=UnitOfTemperature.CELSIUS,
+        state_class=SensorStateClass.MEASUREMENT,
+        translation_key=ThinQProperty.CURRENT_TEMPERATURE,
+    ),
```

Comment:
deivce_class?

Reply:
We use Fahrenheit, Celsius in temperature.
As we know, if we use device_class, we have to use only Celsius. right?